llvm/test/CodeGen/AIE/aie2/hardware-loops/instruction-select-zol-end.mir

```diff
@@ -0,0 +1,27 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+# RUN: llc -O2 -mtriple=aie2 -run-pass=instruction-select %s -verify-machineinstrs -o - | FileCheck %s
+
+---
```

Comment:
A couple of tests here are missing the license & copyright header. Could you please add those in a fixup PR @martien-de-jong?